website/docs/cloud-docs/integrations/kubernetes/index.mdx

```diff
@@ -1,68 +1,163 @@
 ---
-page_title: Terraform Cloud Operator for Kubernetes setup instructions
+page_title: Terraform Cloud Operator for Kubernetes 
 description: >-
   Terraform Cloud Operator allows you to provision infrastructure directly from
   the Kubernetes control plane.
-tfc_only: true
 ---
 
-# Terraform Cloud Operator for Kubernetes Setup Instructions
-
-## Overview
+# Terraform Cloud Operator for Kubernetes Overview
 
 HashiCorp Terraform Cloud customers can integrate with Kubernetes using the official [Terraform Cloud Operator for Kubernetes](https://github.com/hashicorp/terraform-k8s) to provision infrastructure internal or external to the Kubernetes cluster directly from the Kubernetes control plane.  Using the Terraform Cloud Operator for Kubernetes' CustomResourceDefinition (CRD), users can dynamically create Terraform Cloud workspaces using a Terraform configuration from a git repository or from the [Terraform Registry](https://registry.terraform.io/), populate variables, and perform Terraform runs to provision infrastructure.
 
-## Prerequisites
+## Key benefits
```

Comment:
@arybolovlev I think this is the last section we need some info for, just some features to point out that change from v1 to v2

Reply:
We partially have this covered in the FAQ: https://github.com/hashicorp/terraform-cloud-operator/blob/main/docs/faq.md#general-questions but I think more detailed information will be also in the  blog post. If that is OK, could we wait for a few more days to finalize it and then sync here?

Reply:
**What’s new in version 2 of the Terraform Cloud Operator**

The Terraform Cloud Operator has undergone a transformative upgrade from v1 to v2, introducing features that significantly enhance performance, flexibility, and user experience.

- **Flexible resource management:** The Operator now features multiple custom resources each with separate controllers for different Terraform Cloud resources. This provides additional flexibility and the ability to manage more custom resources concurrently, significantly improving performance for large-scale deployments.

- **Namespace management:** The introduction of the --namespace option, allowing users to tailor the Operator’s watch scope to specific namespaces, enabling more fine-grained resource management as setups grow.

- **Configurable synchronization:** With the new --sync-period, synchronization between custom resources and Terraform Cloud is faster than ever, ensuring timely updates and smoother operations.